websites/N/Neko-sama.fr/presence.ts

```diff
@@ -0,0 +1,104 @@
+const presence = new Presence({
+	clientId: "1014298173314961481",
+});
+
+function timeToString(nbr: number): string {
+	let nbrCopy = nbr,
+		nbrString = "",
+		quotient = 0,
+		remainder = 0;
+	if (nbrCopy >= 3600) {
+		quotient = Math.floor(nbrCopy / 3600);
+		if (isNaN(quotient)) quotient = 0;
+		remainder = nbrCopy % 3600;
+		if (quotient > 9) nbrString += `${quotient.toString()}:`;
+		else nbrString += `0${quotient.toString()}:`;
+
+		nbrCopy = remainder;
+	}
+	quotient = Math.floor(nbrCopy / 60);
+	if (isNaN(quotient)) quotient = 0;
+	remainder = nbrCopy % 60;
+	if (quotient > 9) nbrString += `${quotient.toString()}:`;
+	else nbrString += `0${quotient.toString()}:`;
+
+	nbrCopy = remainder;
+	if (isNaN(nbrCopy)) nbrCopy = 0;
+	if (nbrCopy > 9) nbrString += nbrCopy.toString();
+	else nbrString += `0${nbrCopy.toString()}`;
+
+	return nbrString;
+}
+
+interface Video {
+	time: number;
+	duration: number;
+	paused: boolean;
+}
+
+let video: Video = null;
+
+presence.on("iFrameData", (data: Video) => {
+	video = data;
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "nekosama-icon",
```

Comment:
Just someting, but maybe put the text "Navigue sur Neko-sama" here and remove the else at the bottom of the file. It will be the same result and more readable.

Reply:
Thank you for all your suggestions ^^